src/main/java/io/vertx/ext/amqp/impl/AmqpConnectionImpl.java

```diff
@@ -294,16 +294,6 @@ private void configureTheSource(AmqpReceiverOptions receiverOptions, ProtonRecei
     if (receiverOptions.isDurable()) {
       source.setExpiryPolicy(TerminusExpiryPolicy.NEVER);
       source.setDurable(TerminusDurability.UNSETTLED_STATE);
-    } else {
```

Comment:
This only removed the use of the options, not ability to set the options which still remains.

Reply:
Fixed.